static/src/javascripts/lib/cookies.js

```diff
@@ -1,12 +1,16 @@
 // @flow
-const getShortDomain = (): string => {
-    // Trim subdomains for prod (www.theguardian), code (m.code.dev-theguardian) and dev (dev.theguardian, m.thegulocal)
+const getShortDomain = (isCrossSubdomain: ?boolean = false): string => {
```

Comment:
you could use destructuring here so that the order of the args does matter:

`const getShortDomain = ({ isCrossSubdomain = false } = {}): string`

I _think_ this will mean you don't need to type too? thing flow can infer it?